test/bijectors/corr.jl

```diff
@@ -0,0 +1,65 @@
+using Bijectors, DistributionsAD, LinearAlgebra, Test
+using Bijectors: VecCorrBijector, VecCholeskyBijector, CorrBijector
+
+@testset "CorrBijector & VecCorrBijector" begin
+    for d in [1, 2, 5]
+        b = CorrBijector()
+        bvec = VecCorrBijector()
+
+        dist = LKJ(d, 1)
+        x = rand(dist)
+
+        y = b(x)
+        yvec = bvec(x)
+
+        # Make sure that they represent the same thing.
+        @test Bijectors.triu1_to_vec(y) ≈ yvec
+
+        # Check the inverse.
+        binv = inverse(b)
+        xinv = binv(y)
+        bvecinv = inverse(bvec)
+        xvecinv = bvecinv(yvec)
+
+        @test xinv ≈ xvecinv
+
+        # And finally that the `logabsdetjac` is the same.
+        @test logabsdetjac(bvec, x) ≈ logabsdetjac(b, x)
+
+        # NOTE: `CorrBijector` technically isn't bijective, and so the default `getjacobian`
+        # used in the ChangesOfVariables.jl tests will fail as the jacobian will have determinant 0.
+        # Hence, we disable those tests.
+        test_bijector(b, x; test_not_identity=d != 1, changes_of_variables_test=false)
```

Comment:
The `InverseFunctions.test_inverse` that is called in here fails on some calls, but passes on others, even with the same sample `x`. CI on previous commits of this PR failed because of it. Seems weird, not sure why.

Reply:
There was no RNG between these lines, it was after sampling from `LKJ`. I believe the issue was numerical as there was no explicit zero-filling in the old Matrix-version link function, which caused mismatches. I've added zero-filling and tested it a bunch locally, should be fine 🤞